pkgs/applications/emulators/dosbox-staging/default.nix

```diff
@@ -20,22 +25,17 @@
 , ninja
 , opusfile
 , pkg-config
-, irr1
-, SDL2
-, SDL2_image
-, SDL2_net
 , speexdsp
-, stdenv
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (self: {
```

Comment:
By convention we call ``self`` here ``finalAttrs``. We should stick to that for consistency.

Reply:
Oh boy. This convention was attacked precisely here:

https://github.com/NixOS/nixpkgs/pull/212600#discussion_r1086935759